app/api/toc_generation/tocService.ts

```diff
@@ -0,0 +1,46 @@
+import { files, uploadsPath } from 'api/files';
+import { prettifyError } from 'api/utils/handleError';
+import errorLog from 'api/log/errorLog';
+import request from 'shared/JSONRequest';
+import entities from 'api/entities';
+
+const tocService = (serviceUrl: string) => ({
```

Comment:
Is the parenthesis here required? We use this as the shorthand way to return directly an object instead of opening a function, but here it does not appear to be the case?  I am confused.

Reply:
this function actually returns an object, its supposed to be an "instance" method.